api/repositories/service_binding_repository.go

```diff
@@ -142,3 +152,57 @@ func cfServiceBindingToRecord(binding v1alpha1.CFServiceBinding) ServiceBindingR
 		},
 	}
 }
+
+func (r *ServiceBindingRepo) ListServiceBindings(ctx context.Context, authInfo authorization.Info, message ListServiceBindingsMessage) ([]ServiceBindingRecord, error) {
+	nsList, err := r.namespacePermissions.GetAuthorizedSpaceNamespaces(ctx, authInfo)
+	if err != nil {
+		// untested
```

Comment:
Is there any reason to have this branch untested? Why do we need such sort of comments?